### ISSUE TITLE
absolute spinner positioning

### ISSUE THREAD

**Issue by gabrielcsapo** [gabrielcsapo/psychic-ui#16]
current fix from @ppnsanders

```
<div id="loading" class="grid" style="display: none; width: 65%; margin: 0 auto; margin-top: 5%; margin-bottom: 5%;">
    <div class="col-12-12">
        <div class="spinner-overlay">
              <div class="spinner spinner-primary"></div>
        </div>
    </div>
</div>
```

**Comment by gabrielcsapo** [comment]
@ppnsanders this was actually a bug when rendering the spinner

there is a class called `spinner-wrapper` and when used like this

```
div(class="spinner-overlay")
        div(class="spinner-wrapper")
            div(class="spinner spinner-primary")
```

![screen shot 2016-05-12 at 8 27 33 pm](https://cloud.githubusercontent.com/assets/1854811/15237107/fcae2c08-187f-11e6-9c53-ea28d1fc88af.png)

**Comment by gabrielcsapo** [comment]
I will update the code examples and fix a css bug that was causing this.

**Comment by gabrielcsapo** [comment]
code has been merged in #17 and is now live at http://gabrielcsapo.github.io/psychic-ui/ 😄

**Comment by ppnsanders** [comment]
Thanks @gabrielcsapo!
